deformable_ompl/ComputeTrialSetupQuery query
---
deformable_ompl/ComputeTrialSetupResult result
